### PR TITLE
Add Sprint 5 detailed priorities to PROJECT_PLAN.md

### PR DESCRIPTION
**Pull request by jeffreyhorn**
Based on Sprint 4 Retrospective recommendations and research findings.

Sprint 5 Priorities (10 days):

Priority 1: Fix Min/Max Reformulation Bug (1-2 days)
- Issue: Auxiliary constraint multipliers not in stationarity equations
- Research: docs/research/minmax_objective_reformulation.md
- Solution: Strategy 2 (Direct Constraints) for objective-defining cases
- Test Cases: 5 comprehensive cases from research doc
- Related: src/kkt/reformulation.py, src/kkt/assemble.py

Priority 2: Complete PATH Solver Validation (1 day)
- Status: Test framework exists, GAMS/PATH now available
- Tasks: Execute validation suite, investigate Model Status 5 failures
- Files: tests/validation/test_path_solver.py, test_path_solver_minmax.py

Priority 3: Production Hardening (2-3 days)
- Error recovery and graceful handling
- Large model testing (1000+ vars, 10000+ eqs)
- Memory optimization and profiling
- Edge case testing (20+ new tests)

Priority 4: PyPI Packaging (1-2 days)
- Package configuration and wheel building
- Release automation (GitHub Actions)
- Multi-platform testing (Linux, macOS, Windows)
- Version 0.4.0 → 1.0.0 release

Priority 5: Documentation Polish (2 days)
- Tutorial for beginners
- FAQ section
- Enhanced troubleshooting guide
- API reference (Sphinx)

Updated existing Sprint 5 sections with notes on what's already complete from Sprint 4 (CI/CD, user guide, examples).

Total: ~190 lines of detailed planning added
Structure: 5 priorities with implementation steps, test cases, acceptance criteria, and deliverables

Based on: docs/planning/SPRINT_4/RETROSPECTIVE.md recommendations
Research: docs/research/minmax_objective_reformulation.md
Context: Post Sprint 4 completion, ready for Sprint 5 kickoff

### CODE REVIEW

File: docs/planning/PROJECT_PLAN.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ Here’s a concrete five-sprint (2 weeks each) plan to build **nlp2mcp**, a Pyth
 
 * **Includes & parameters:**
 
-  * Support `$include` for simple parameter files (string-substitute, no conditionals).
+  * Support `$include` fo 3e parameter files (string-substitute, no conditionals).
```
[the remark this answers is not in the document]
Comment (jeffreyhorn) [reply docs/planning/PROJECT_PLAN.md]: fixed.
